src/screens/DetailsScreen.js

```diff
@@ -36,17 +39,14 @@ const DetailsScreen = () => {
 
                 <View style={styles.horizontalView}>           
                     <Text style={styles.subtitle}>FOUNDER</Text>
-                    <Text style={styles.text}>Founder example</Text>
+                    <Text style={styles.text}>{founder ? founder.name : "Unknown"}</Text>
                 </View>
                 <View style={styles.horizontalView}>
                     <Text style={styles.subtitle}>WORDS</Text>
-                    <Text style={styles.text}>Words example</Text>
+                    <Text style={styles.text}>{house.words ? house.words : "Unknown"}</Text>
```

Comment:
Uma dica, nesses casos de operador ternário onde o primeiro e segundo elementos são iguais, ou seja `x ? x : y`, dá para utilizar o operador `??` do javascript que diz basicamente que se o primeiro elemento existir retorne ele, se não, retorne o segundo. No seu caso ficaria assim: `house.words ?? "Unkown"`.
Mas não é nada que vá mudar a funcionalidade, é só uma dica pra usar futuramente

Reply:
Com existir eu quero dizer se ele retorna true, ou seja, se `!!primeiroElemento`

Reply:
Por algum motivo, os resultados estão sendo diferentes. Se eu usar `house.words ? house.words : "Unknown"` e o dado for `""`, aparecerá `"Unknown"`. Mas se eu usar `house.words ?? "Unknown"` ele não exibe o resultado (provavelmente porque está considerando `house.words` como um valor válido, ainda que seja uma string sem caracteres).